source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -115,43 +116,124 @@ - (UIView *)render
     newView.backgroundColor = [_hostConfig getBackgroundColorForContainerStyle:
         [ACOHostConfig getPlatformContainerStyle:style]];
 
-    auto backgroundImageProperties = [_adaptiveCard card]->GetBackgroundImage();
+    [ACRView renderBackgroundImageView:[_adaptiveCard card]->GetBackgroundImage().get() containerView:newView rootView:self];
+    
+    [self callDidLoadElementsIfNeeded];
+    return newView;
+}
+
++ (void)renderBackgroundImageView:(const AdaptiveCards::BackgroundImage *)backgroundImageProperties containerView:(UIView *)containerView rootView:(ACRView *)rootView
```

Comment:
Can you add this function to Util.h? ACRView module is getting bloated, and this function  doesn't look like it needed to be part of a class.